apps/fabric-website/src/pages/Styles/TypographyPage/docs/web/TypographyCustomization.md

```diff
@@ -0,0 +1,29 @@
+You can add some of your own customizations to the default theme.
+For example, you can set the `defaultFontStyle` property to modify every font variant
+(small, medium, large, etc.), or you can target specific variants through the `fonts` property.
+These can be used separately, or together, as shown in the example below.
+The overrides can include any property from `IRawStyle`.
```

Comment:
This is great @natalieethell ! One small suggestion: it feels it'd be helpful to link directly to the definition of `IRawStyle` here, e.g.
```suggestion
The overrides can include any property from [`IRawStyle`](#/controls/web/references/irawstyle).
```

Thoughts?

Reply:
That makes sense to me!